### ISSUE TITLE
Namespace

### ISSUE THREAD

**Issue by nc** [stitchesjs/stitches#1057]
**Is your feature request related to a problem? Please describe.**
I have stitches being used by two different component libraries on the same page, and they are clashing.

**Describe the solution you'd like**
A way to define a namespace. Like a theme, but for isolating themes too.

**Describe alternatives you've considered**
Namespacing variables with `$libName$accent$1` etc. A bit too wordy.

**Additional context**
N/A

**Comment by KXLAA** [comment]
You can prefix the CSS variable names in your `createStitches` function. I think this should solve clashing styles

For example: 

`const { theme } =  createStitches({ prefix: "your-prefix", theme: {...} })`

see more [here](https://stitches.dev/docs/api)

**Comment by hadihallak** [comment]
As @KXLAA mentioned, a prefix is already supported.
